js/repeater.js

```diff
@@ -603,7 +609,13 @@
 								if(index<renderer.nested.length){
 									loopNested(cont, index);
 								}else{
-									proceed('complete', args);
+									if (++skip % 50 === 0) {
```

Comment:
We should make this value a configurable option so that this can be optimized best for individual repeater performance. Perhaps a name like `recursionLimit' or something along those lines